### PR TITLE
Add tier 1 clusters for VSC UGent

### PR DESCRIPTION
**Pull request by nvnieuwk**

### CODE REVIEW

**Comment by maxulysse** [comment]
Don't you need to add this tier1_project params to the schema ignore?

**Comment by nvnieuwk** [comment]
Good point! I'll add it, but it should be a required parameter when running with the tier1 clusters so I also have to find a way to enforce that

**Comment by maxulysse** [comment]
> Good point! I'll add it, but it should be a required parameter when running with the tier1 clusters so I also have to find a way to enforce that

cf https://github.com/nf-core/configs/blob/8105c09b61c9312a7a78015098a953b862de14d7/conf/uppmax.config#L9

**Comment by nvnieuwk** [comment]
@maxulysse tests are still failing because the error message produced by the config, what should I do about this?

File: conf/vsc_ugent.config [hunk 1]
```diff
@@ -4,6 +4,19 @@ if (!params.tier1_project) {
     params.tier1_project = null
 }
 
+// Get the hostname and check some values for tier1
+def hostname = "doduo"
```
Comment (maxulysse) [inline conf/vsc_ugent.config:8]: pokemon fan?

**Review by maxulysse — APPROVED** [review]

File: conf/vsc_ugent.config (hunk 1 above)
Comment (nvnieuwk) [reply conf/vsc_ugent.config:8]: Hehehehe maybe ;) all our clusters are named after pokemon

Comment (maxulysse) [reply conf/vsc_ugent.config:8]: yeah, I noticed dodrio too

**Pull request merged**